zerver/lib/webhooks/common.py

```diff
@@ -86,6 +87,9 @@ def check_send_webhook_message(
     ),
     unquote_url_parameters: bool = False,
 ) -> None:
+    if all_event_types is not None and complete_event_type not in all_event_types:
+        raise JsonableError(f'Event "{complete_event_type}" does not present in all_event_types')
```

Comment:
Hmm, do we want this to be a runtime error, or just a test error only?  I think the latter makes more sense with our current model, unless we want to raise `UnexpectedWebhookEventType`.  

(Third-party services add new event types from time to time, and we don't want to have this code reject such events in production, when the consequence of it not being listed would just be having the event type not documented)

Reply:
I'm not sure what a test error would be. Then should we just use an assertion here?